code/modules/mob/living/carbon/human/stripping.dm

```diff
@@ -75,9 +75,11 @@
 
 	if(!do_after(user,HUMAN_STRIP_DELAY,src))
 		return
-
-	if(!stripping && user.get_active_hand() != held)
+if(!stripping)
+	if(user.get_active_hand() != held)
 		return
+	if(held == loc)
+		return // No recursion

```

Comment:
```suggestion
	if(!stripping)
		if(user.get_active_hand() != held)
			return
		if(held == loc)
			return // No recursion

```

Reply:
What? held == loc? This doesn't even work. Loc would be the person holding the micro in this case. Also, why get rid of the message? People need to know that trying to break reality is unacceptable.

Reply:
Micros do not get directly held. The micro's actual `loc` is the `/obj/item/weapon/holder` that they're contained within, which is in turn contained within the user. I just forgot to add the message.